src/Domain/Entities/Servers/Enums/ServerIdentifier.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Crpg.Domain.Entities.Servers.Enums;
+public enum ServerIdentifier
+{
+    Crpg01,
+    Crpg02,
+    Crpg03,
+    // ... other server identifiers ...
+}
```

Comment:
Goes up to 4.

For each server there is a letter
Crpg01a
Crpg01b
Crpg01c
Crpg01d
Crpg01e

a is battle
b is siege
c is duel
d is skirmish
e is DTV